actors/builtin/miner/monies.go

```diff
@@ -64,15 +70,20 @@ func PledgePenaltyForTermination(initialPledge abi.TokenAmount, sectorAge abi.Ch
 // newly-committed power, holding the per-epoch block reward constant (though in reality it will change over time).
 // The network total pledge and circulating supply parameters are currently unused, but may be included in a
 // future calculation.
-func InitialPledgeForPower(qaPower abi.StoragePower, networkQAPower abi.StoragePower, networkTotalPledge abi.TokenAmount, epochTargetReward abi.TokenAmount, networkCirculatingSupply abi.TokenAmount) abi.TokenAmount {
+func InitialPledgeForPower(qaPower abi.StoragePower, networkQAPower, targetPower abi.StoragePower, networkTotalPledge abi.TokenAmount, epochTargetReward abi.TokenAmount, networkCirculatingSupply abi.TokenAmount) abi.TokenAmount {
 	// Details here are still subject to change.
 	// PARAM_FINISH
 	// https://github.com/filecoin-project/specs-actors/issues/468
-	_ = networkCirculatingSupply // TODO: ce use this
-	_ = networkTotalPledge       // TODO: ce use this
+	_ = networkTotalPledge // TODO: ce use this
 
 	if networkQAPower.IsZero() {
 		return epochTargetReward
 	}
```

Comment:
> I think we should push this check into ExpectedDayRewardForPower, next to the division it is protecting. Then ipBase will more correctly become 20*BR instead of, currently, 1BR.

👍 

> Logically I think we want this to come out as PledgeShare=1, which would be achieved with storageToCover=sectorQAPower. This suggests a max(sectorQAPower, networkPower, baselinePower) for storageToCover.

👍 

> This begs the question of how do we actually want to compute pledge for power in the genesis state, which I think we can address head-on separately, and then update here accordingly for networkQAPower=0.

This sounds out of scope of this PR. Should I file an issue for this here?